scout/server/blueprints/variants/forms.py

```diff
@@ -123,6 +123,8 @@ class VariantFiltersForm(FlaskForm):
     filter_variants = SubmitField(label="Filter variants")
     export = SubmitField(label="Filter and export")
 
+    hide_unaffected = BooleanField("Hide variants present only in unaffected", default=False)
```

Comment:
I think it would make sense to leave this on by default.

```suggestion
    hide_unaffected = BooleanField("Hide variants present only in unaffected", default=True)
```
One could restate in the positive, like `Show variants only present in unaffected`, but maybe it looks odd together with 'Hide dismissed'?

Reply:
I agree that having it on by default is the way to go. Yesterday I've tried exactly the same code in your suggestion and didn't work, let's try today!

Reply:
Nope, doesn't work. I'll try to fix it in another way

Reply:
I think I'll just change the label of the checkbox into "Show variants present only in unaffected" and change the query from $nin to $in, much easier this way. Also because when you land into the variantS page (any type= with the GET method) then the query for filtering out these vars doesn't contain hide_unaffected=True and to add it for each type of var is cumbersome..